tests/basic/storage_pool_test.go

```diff
@@ -6315,3 +6315,88 @@ var _ = Describe("{PoolResizeInvalidPoolID}", func() {
 		AfterEachTest(contexts, testrailID, runID)
 	})
 })
+
+var _ = Describe("{PoolResizeReduceErrorcheck}", func() {
+	var testrailID = 79487
+	// Testrail Description : Resize with invalid pool size less than existing
+	// Testrail Corresponds : https://portworx.testrail.net/index.php?/cases/view/84470
```

Comment:
please check if the testrail description and corresponds needs to be updated ?

Reply:
this i have to check with leela

Reply:
Done